cluster/plan.go

```diff
@@ -236,21 +270,27 @@ func (c *Cluster) BuildKubeControllerProcess(prefixPath string) v3.Process {
 	}
 
 	CommandArgs := map[string]string{
-		"address":                          "0.0.0.0",
-		"cloud-provider":                   c.CloudProvider.Name,
+		"address":                          "127.0.0.1",
 		"allow-untagged-cloud":             "true",
+		"allocate-node-cidrs":              "true",
+		"cloud-provider":                   c.CloudProvider.Name,
+		"cluster-cidr":                     c.ClusterCIDR,
 		"configure-cloud-routes":           "false",
-		"leader-elect":                     "true",
-		"kubeconfig":                       pki.GetConfigPath(pki.KubeControllerCertName),
 		"enable-hostpath-provisioner":      "false",
+		"kubeconfig":                       pki.GetConfigPath(pki.KubeControllerCertName),
+		"leader-elect":                     "true",
 		"node-monitor-grace-period":        "40s",
 		"pod-eviction-timeout":             "5m0s",
-		"v":                                "2",
-		"allocate-node-cidrs":              "true",
-		"cluster-cidr":                     c.ClusterCIDR,
-		"service-cluster-ip-range":         c.Services.KubeController.ServiceClusterIPRange,
-		"service-account-private-key-file": pki.GetKeyPath(pki.ServiceAccountTokenKeyName),
+		"profiling":                        "false",
 		"root-ca-file":                     pki.GetCertPath(pki.CACertName),
+		"service-account-private-key-file": pki.GetKeyPath(pki.ServiceAccountTokenKeyName),
+		"service-cluster-ip-range":         c.Services.KubeController.ServiceClusterIPRange,
+		"terminated-pod-gc-threshold":      "1000",
```

Comment:
can you explain the main reason why is this 1000 instead of the default 12500

Reply:
Here's the CIS Benchmark justification:

###1.3.1 Ensure that the --terminated-pod-gc-threshold argument is set as appropriate (Scored)

####Description:
Activate garbage collector on pod termination, as appropriate.

####Rationale:
Garbage collection is important to ensure sufficient resource availability and avoiding
degraded performance and availability. In the worst case, the system might crash or just be
unusable for a long period of time. The current setting for garbage collection is 12,500
terminated pods which might be too high for your system to sustain. Based on your system
resources and tests, choose an appropriate threshold value to activate garbage collection.

####References:
1. https://kubernetes.io/docs/admin/kube-controller-manager/
2. https://github.com/kubernetes/kubernetes/issues/28484

####CIS Controls:
Version 6

14 Controlled Access Based on the Need to Know
Controlled Access Based on the Need to Know